vagrant/README.md

```diff
@@ -124,3 +124,11 @@ Now start things up, but specify the aws provider:
 Your instances should get tagged with a name including your hostname to make
 them identifiable and make it easier to track instances in the AWS management
 console.
+
+### Libvirt with kvm ###
+
+Install the `vagrant-libvirt` plugin to provide libvirt and kvm support:
+```vagrant plugin install vagrant-libvirt``` or ```yum install vagrant-libvirt``` for fedora >=21
```

Comment:
Maybe this is more of an issue with libvirt, but I had a heck of a time even getting basic Virtualbox commands to run because it looks like the dependencies aren't setup properly in the gems. Can we get a complete set of commands that actually works now? We can always follow up with a clean up later when it can be reduced to just a couple of commands.